<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkRemoteSettingsPage</class>
 <widget class="QWidget" name="NetworkRemoteSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote</string>
  </property>
  <layout class="QVBoxLayout" name="layout_networkremotesettingspage">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="useRemoteClient">
       <property name="text">
        <string>Use Remote Network Client</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="settingsBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="remotePortLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Remote Port</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="portSelected">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>71</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="localConnectionsOnly">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>231</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Only allow local connections</string>
        </property>
       </widget>
       <widget class="QLabel" name="ipAddressLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Your IP Address is</string>
        </property>
       </widget>
       <widget class="QLabel" name="ip_address">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>110</y>
          <width>191</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
